src/box/space.h

```diff
@@ -269,6 +269,14 @@ space_is_temporary(struct space *space)
 	return space->def->opts.is_temporary;
 }
 
+/** Return true is space is ephemeral */
+static inline bool
+space_is_ephemeral(struct space *space)
```

Comment:
Why did you add this function?

Reply:
Remove it, i use it to check that there is no compression for ephemeral spaces, but i think  this is unnecessary